docs/platform/known-issues.md

```diff
@@ -0,0 +1,36 @@
+---
+description: known issue in the Grid Platform
+---
+
+# Known Issues
+#### Last Updated: April 1, 2022
+
+### [BYOC](https://docs.grid.ai/platform/custom-cloud-credentials/byoc)
+* Creating runs with the same name across two cluster is currenly supported, but prohibits certain operations against runs/experiments. 
+
+### [Artifacts](https://docs.grid.ai/features/runs/artifacts)
+* Artifacts don't sync for fast experiments: We've detected a race condition with short-running experiments, which may cause artifacts not to be properly synced. We recommend ensuring your experiments last at least a minute (to be safe), and sleep if needed as a workaround. 
+* Canceling download of Artifacts: When downloading the artifacts, Ctrl-C does not cancel the download. As a workaround, use Ctrl-Z followed by `kill -9 $(jobs -p)`.
+* Artifacts are not saved in a subdirectory: Grid artifacts are not saved in a subdirectory for run name and experiment name. This will overwrite files if two experiments have artifacts with the same file name. Workaround would be to put the artifacts in separate folders before downloading artifacts again.
```

Comment:
not sure what this means. can you clarify? we want to make sure this is clear from a user perspective

Reply:
There are 3 different know issues with Artifacts 
1. Artifacts don't sync for fast experiments
2. Canceling download of Artifacts
3. Artifacts are not saved in a subdirectory